dotcom-rendering/src/components/DateTime.tsx

```diff
@@ -9,9 +9,18 @@ type Props = {
 	showWeekday: boolean;
 	showDate: boolean;
 	showTime: boolean;
-	display?: 'absolute' | 'relative';
 };
 
+type DisplayProps =
+	| {
+			display?: 'absolute';
+			absoluteServerTimes?: never;
+	  }
+	| {
+			display: 'relative';
+			absoluteServerTimes: boolean;
+	  };
+
```

Comment:
This is the only real change.

When a time is displayed as “relative”, the consumer must decide whether the time rendered on the server should be relative also.